content/integrations/haproxy.md

```diff
@@ -3,17 +3,50 @@ title: Datadog-HAProxy Integration
 integration_title: HAProxy
 kind: integration
 git_integration_title: haproxy
+updated_for_agent: 5.8.5
+newhlevel: true
 ---
 
-### Overview
+# Overview
+
+![HAProxy default dashboard](/static/images/haproxydash.png)
 
 Capture HAProxy activity in Datadog to:
 
 * Visualize HAProxy load-balancing performance.
 * Know when a server goes down.
 * Correlate the performance of HAProxy with the rest of your applications.
 
-![HAProxy default dashboard](/static/images/haproxy.png)
+# Installation
+
+Make sure that stats are enabled on your HAProxy configuration
```

Comment:
Can we be more specific here? Either provide the directives for this or link to the haproxy docs.